src/test/java/serviceimpl/FileWriterImplTest.java

```diff
@@ -0,0 +1,39 @@
+package serviceimpl;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import service.FileWriter;
+
+public class FileWriterImplTest {
+    private static FileWriter fileWriter;
+    private static final String WAY_TO_REPORT = "src/main/resources/CloseDay.csv";
+    private static String expectedString;
+
+    @BeforeClass
+    public static void beforeClass() {
+        expectedString = "fruits, quantity" + System.lineSeparator()
+                + "banana" + "," + "152" + System.lineSeparator()
+                + "apple" + "," + "90";
+    }
+
+    @Test
+    public void fileWriter_Ok() throws IOException {
+        String expected = expectedString;
+        String actual = Files.readString(Path.of(WAY_TO_REPORT));
```

Comment:
Make a separate method for reading, and use it here. And also don't throw an exception in that method, but catch it and throw your exception